### PR TITLE
Sub branch 1 1

### PR DESCRIPTION
**Pull request by ArvindKarthi**

### CODE REVIEW

File: src/components/form/elements/textField/index.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
+import React from "react";
+
+const TextField = ({ attributes }) => {
+  const { type, label, size, placeholder } = attributes;
+  return (
+    <div className="form-group">
+      <label htmlFor={label} className={`d-block fn-${size} form-label`}>
```
Comment (achyuthgroot) [inline src/components/form/elements/textField/index.js:7]: Shall we move label as a separate component ?

File: src/config/index.js [hunk 2]
```diff
@@ -0,0 +1,26 @@
+// attributes for each text fields that would make each textField component different.
+const textFieldAttributes = [
+  { label: "Username", size: "small", placeholder: "fullname", type: "text" },
+  {
+    label: "Password",
+    size: "small",
```
Comment (achyuthgroot) [inline src/config/index.js:6]: Follow some standard here

File: src/components/form/elements/textField/index.js (hunk 1 above)
Comment (ArvindKarthi) [reply src/components/form/elements/textField/index.js:7]: Yeah bro will do and update the code.

File: src/config/index.js (hunk 2 above)
Comment (ArvindKarthi) [reply src/config/index.js:6]: Ok bro.

**Pull request merged**